### PR TITLE
Cpu hotplug: Introduce test for default topology

### PR DESCRIPTION
**Pull request by xpivarc**
### What this PR does
This test exercise a path where a topology is not provided instead we use the requests. This covers a regression found in 1.2 release.

Fixes #

Links to places where the discussion took place: 

### Release note

```release-note
NONE
```

### CODE REVIEW

File: tests/hotplug/cpu.go [hunk 1]
```diff
@@ -60,6 +60,23 @@ var _ = Describe("[sig-compute][Serial]CPU Hotplug", decorators.SigCompute, deco
 
 	})
 
+	Context("with requests without topology", func() {
+
+		It("should be able to start", func() {
+
+			vmi := libvmifact.NewAlpineWithTestTooling(
+				libnet.WithMasqueradeNetworking(),
```
Comment (orelmisan) [inline tests/hotplug/cpu.go:68]: Does having a network connection somehow affect the test?

Comment (xpivarc) [reply tests/hotplug/cpu.go:68]: No, copy-paste for this one.

Comment (orelmisan) [reply tests/hotplug/cpu.go:68]: Please consider removing it.

Comment (xpivarc) [reply tests/hotplug/cpu.go:68]: I'll update the PR later.

**Comment by xpivarc** [comment]
> /hold This should now fail properly.

/hold cancel
Actually I meant to open a new PR for the breaking test. This should be ready now

**Review by fossedihelm — APPROVED** [review]

**Pull request merged**